hdate/date.py

```diff
@@ -205,6 +208,82 @@ def omer_day(self):
             return 0
         return omer_day
 
+    @property
+    def next_day(self):
```

Comment:
No idea if this is possible, but maybe we could make HDate an iterable, allowing us to use next() builtin

Reply:
Wouldn't really make sense as an iterable since there's no real end. You'd be better off with a while loop with some arbitrary condition.

Reply:
Not really sure it doesn't make sense. You definitely can have an iterable with no end.

It simply allows you advance indefinitely without using more memory.